sentinelsat/sentinel.py

```diff
@@ -587,44 +614,168 @@ def download_all(self, products, directory_path='.', max_attempts=10, checksum=T
             A dictionary containing the return value from download() for each successfully
             downloaded product.
         dict[string, dict]
-            A dictionary containing the product information for products whose retrieval
-            from the long term archive was successfully triggered.
-        set[string]
-            The list of products that failed to download.
+            A dictionary containing the product information for products successfully
+            triggered for retrieval from the long term archive but not downloaded.
+        dict[string, dict]
+            A dictionary containing the product information of products where either
+            downloading or triggering failed
         """
+
+        lta_retry_delay = 600 # try to request a new product from the LTA every 600 seconds
```

Comment:
Do we need to expose this as a function parameter or class attribute?

Reply:
Passed as a function parameter